pkg/grpc/api.go

```diff
@@ -1263,7 +1263,13 @@ func (a *api) GetConfigurationAlpha1(ctx context.Context, in *runtimev1pb.GetCon
 		Metadata: in.Metadata,
 	}
 
-	getResponse, err := store.Get(ctx, &req)
+	policy := a.resiliency.ComponentPolicy(ctx, in.StoreName)
+	getResponse := &configuration.GetResponse{}
```

Comment:
You want `var getResponse *configuration.GetResponse` here to avoid allocating a struct that is never used.